casual/casual-event-api/src/main/java/se/laz/casual/event/ServiceCallEvent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2024, The casual project. All rights reserved.
+ *
+ * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+ */
+package se.laz.casual.event;
+
+import javax.transaction.xa.Xid;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface ServiceCallEvent
```

Comment:
Quick blurb javadoc detailing what this even is.

Reply:
Done